<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostNAS_ConfDialogBase</class>
 <widget class="QDialog" name="PostNAS_ConfDialogBase">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PostNAS-Suche</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Datenbankeinstellungen</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Datenbankname</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Schema</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="auth_2">
         <attribute name="title">
          <string>Authentifizierung</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Benutzername</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leUID">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Passwort</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePWD">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leSCHEMA">
        <property name="text">
         <string>public</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leHOST">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lePORT">
        <property name="inputMask">
         <string>99999</string>
        </property>
        <property name="text">
         <string>5432</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leDBNAME"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="testConnectionButton">
        <property name="text">
         <string>Verbindung testen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Zugriffssteuerung</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="4">
       <widget class="QTableWidget" name="userTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Benutzername</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name, Vorname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Berechtigung</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="accessControlButtonAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hinzufügen</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="accessControlButtonEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Bearbeiten</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="accessControlButtonDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Löschen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Aktivieren</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
